exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -111,16 +91,16 @@ impl JavaServiceRuntime {
     /// Loads an artifact from the specified location involving verification of the artifact.
```

Comment:
>  pub fn create_service(&self, artifact_id: &str) -> ServiceProxy {

Why doesn't it return a Result? This operation can easily fail in Java.

Reply:
It's not trivial to deal with Result here because of different types of errors that could happen. Thus for the moment it just panics for error cases, which looks pretty natural for the current level of support of multiple services, but probably not the best solution for dynamic services.